doc/elements/document.schema.json

```diff
@@ -22,6 +22,10 @@
         "pattern": "^\\d+\\.\\d+(\\.\\d+)?$"
       }
     },
+    "doi": {
+      "type": "string",
+      "pattern": "^10\\.5594\\/SMPTE\\.?[A-Za-z\\d-\\/.]+$"
```

Comment:
Pattern should be looser

Reply:
My bad, made this change before talking to you, forgot about it. Updated in https://github.com/SMPTE/document-repo-spec/pull/17/commits/4312a3f515f1de1c9a32ba08969861a142451845 to be the same as the others.